components/threading/virtualized_thread_viewer/create_comment.tsx

```diff
@@ -68,7 +70,10 @@ const CreateComment = forwardRef<HTMLDivElement, Props>(({
             <div className='channel-archived-warning__container'>
                 <BasicSeparator/>
                 <div className='channel-archived-warning__content'>
-                    <i className='icon icon-archive-outline'/>
+                    <ArchiveOutlineIcon
+                        size={20}
+                        color={'rgba(var(--center-channel-color-rgb), 0.56)'}
```

Comment:
@thinkGeist was `currentColor ` not working as intended

Reply:
I actually forgot to try that adjustment, I'll give it a shot and push it up if all looks well.

Reply:
@AshishDhama 

Seems like I **cannot** make the adjustment here unless we want to lose the alpha channel since there isn't a way to break `currentColor` apart and add an alpha channel like what's being used here.

Here's what it looks like with the Alpha channel preserved on Dark/Light themes:
![image](https://user-images.githubusercontent.com/9254315/174853697-bc918145-3bb9-472c-bc2f-7df8f85c967a.png)
![image](https://user-images.githubusercontent.com/9254315/174853733-14c499e0-657b-4b94-8eb9-3ae0f8493f68.png)

If we switch to `currentColor` here and drop the alpha channel:
![image](https://user-images.githubusercontent.com/9254315/174853935-9f35cf56-7d6d-43dc-8c99-d09d2dafe07a.png)
![image](https://user-images.githubusercontent.com/9254315/174854009-ba9f6c75-b97c-4e43-a263-c012d4da5db4.png)

If we're okay with that change then I'll use `currentColor` and drop the alpha channel here, if we want to retain it I'll leave this instance of `color` alone, and only make the adjustment on `searchable_channel_list`

Reply:
@thinkGeist Make sense, then let's go with your current solution, thanks for trying it.